### DISCUSSION TITLE
How to enable PV snapshots under OpenShift in CRC

### DISCUSSION THREAD

**Discussion by texkelly** [crc-org/crc#3960] [Q&A]
It appears that OpenShift in CRC provisions PVs using local host bind mounts. This is not amenable to snapshotting, which we would like to be able to do for the DB. If CRC can supply the option of attaching a second virtio-blk disk we could install the OpenShift LVM operator and use LVM snapshots. Unless someone has a better idea? I'd rather not have to take a copy of the entire crc disk image.

**Comment by praveenkumar** [comment]
@texkelly we have to check if it is possible for all the different platform, may be create a feature request so that we can track?
